backend/app/api/v1/endpoints/conversations.py

```
@@ -0,0 +1,38 @@
+        raise HTTPException(status_code=404, detail="Conversation not found")
+    return db_conversation
+
+@router.put("/{conversation_id}", response_model=Conversation)
+def update_existing_conversation(conversation_id: UUID, title: str, db: Session = Depends(get_db)):
+    db_conversation = update_conversation(db=db, conversation_id=conversation_id, title=title)
+    if db_conversation is None:
+        raise HTTPException(status_code=404, detail="Conversation not found")
+    return db_conversation
+
+@router.delete("/{conversation_id}")
+def delete_existing_conversation(conversation_id: UUID, db: Session = Depends(get_db)):
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)